package.json

```diff
@@ -77,7 +77,7 @@
     "node": ">=8.8.x"
```

Comment:
`>=8.8.x` -> `^12` because yargs and Node 10 EOL

Reply:
@jan-molak Thanks for posting that! Will do this.

Reply:
I've updated Node to min 12, but CircleCI should be also adjusted to this.
Here is PR from Keen https://github.com/angular/protractor/pull/5518